recipes/cn-cbor/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "20200326":
+    sha256: 7cc228a7ac56494f95b74dcf94c254f920aada736793cdf647f9a644f6c67c66
+    url: https://github.com/jimsch/cn-cbor/archive/e2b410487ab76a1b9ee639716c01622290a46dcb.zip
```

```suggestion
    url: https://github.com/jimsch/cn-cbor/archive/277e105f3ac52223408d35c25e01c0ddd0de217f.zip
```

Use the latest version available

---

no, the latest version available is not in the master branch, but in the complete branch...
the master branch is basically the same as https://github.com/cabo/cn-cbor

---

oh boy ... thanks for the information, now it explains all these things.